python-sdk/nuscenes/eval/detection/tests/test_algo.py

```diff
@@ -49,10 +49,10 @@ def random_attr():
 
                 this_gt.append(EvalBox(
                     sample_token=str(sample_itt),
-                    translation=list(np.random.rand(2)*15) + [0],
-                    size=list(np.random.rand(3)*4),
-                    rotation=list(np.random.rand(4)),
-                    velocity=list(np.random.rand(3)*4),
+                    translation=tuple(list(np.random.rand(2)*15) + [0.0]),
+                    size=tuple(np.random.rand(3)*4),
+                    rotation=tuple(np.random.rand(4)),
+                    velocity=tuple(np.random.rand(3)[:2]*4),
```

Comment:
This looks wonky, but it's to preserve the test fixture (random number generator)